src/components/TimerDisplay.js

```diff
@@ -0,0 +1,11 @@
+import React, { useState } from 'react';
+
+const TimerDisplay = (props) =>{   
+    return(
+        <div>
+            Time spent: {props.timeElapsed}
+        </div>
+    )
+}
```

Comment:
```suggestion
const TimerDisplay = ({timeElapsed}) => (
        <div>
            Time spent: {timeElapsed}
        </div>
    )
```

Try to make code as much DRY as you can. :)

Reply:
Możesz mi proszę powiedzieć co narusza tę zasadę? Rozumiem, że skoro ten komponent w swojej wersji zwraca tylko jednego diva ze zemitowaną wartością, to można to uprościć tak jak opisałeś, ale w którym miejscu tam się powtarzam?

Reply:
Chodzi o uproszczenie kodu, gdybyś chciał stworzyć nowy komponent na podstawie tego i w tym celu wykorzystał kopiuj/wklej, to byś miał już powtarzający się zbędny kod, stąd też komentarz o DRY.